src/interfaces/taskDetails.type.ts

```diff
@@ -27,4 +27,5 @@ export type TextAreaProps = {
     name: string;
     value: string;
     onChange: ChangeEventHandler;
+    testId: string;
```

Comment:
Same, why are we having this as a required prop?

Reply:
Mentioned Here https://github.com/Real-Dev-Squad/website-status/pull/663/files/a79a626cd19b3ff9cda85a127be6523e1324b53d#r1263700632